### DISCUSSION TITLE
Oxia bare metal installation

### DISCUSSION THREAD

**Discussion by Nictec** [oxia-db/oxia#539] [Q&A] [answered]
Hello,
is there a reason why the bare metal installation is discouraged in the docs? I would like to use oxia without Kubernetes and don´t see a technical reason why storing the coordinators metadata in a file is worse than in ConfigMap. It would help if somebody could explain to me what I am missing. Thanks in adavance.

**Comment by merlimat** [comment, marked as answer]
There are 2 config maps in typical deployment: 
 1. Configuration (list of namespaces, list of servers with URLs)
 2. `coordinator-status` where the cluster status is checkpointed
 
 (1) This can be just read from a local file. It's just configuration
 
 (2) This is critical information that most not be lost. The `term` of a shard cannot be reset to an earlier value, or it will lead to data corruption issues.

In a bare metal deployment the cluster-checkpoint would have to stored and replicated to multiple nodes.

**Reply by Nictec** [reply]
Makes sense, thanks for the answer. In my usecase we have an existing ceph cluster. For that reason I am going to evaluate adding a ceph metadata store to the coordinator myself.

**Reply by merlimat** [reply]
If you have Ceph mounted as local volume, you should be good just using the local file approach for the status.
